docs/pages/pack/docs/features/css.mdx

```diff
@@ -50,6 +52,11 @@ Using the CSS `@import` syntax to import other CSS files works **out-of-the-box*
 @import './dark.css';
 ```
 
+<Callout type="info">
+  Turbopack does not currently support absolute uri imports in css, such as `@import(https://example.com/stylesheet.css)`. As a workaround,
+  [add `<link href="https://example.com/stylesheet.css"></link>` to `pages/_document.js`](https://nextjs.org/docs/messages/no-page-custom-font#possible-ways-to-fix-it).
+</Callout>
```

Comment:
This is Next.js specific advice, right? Maybe "To work around this in Next.js, " instead? Also did I hear this advice only works in pages?